pkg/combustion/templates/k3s-multi-node-installer.sh.tpl

```diff
@@ -38,6 +38,28 @@ CONFIGFILE={{ .configFilePath }}/{{ .initialiserConfigFile }}
 mkdir -p /opt/eib-k8s/manifests
 cp {{ .manifestsPath }}/* /opt/eib-k8s/manifests/
 
+cat <<- 'EOF' > /opt/eib-k8s/create_manifests.sh
```

Comment:
I'm wondering if we can extract this as a template. It's pretty much the same across all files with the only difference being the path to the Kubernetes config. Would it be too much work?